tasks/main.yml

```diff
@@ -1,5 +1,18 @@
 ---
+## Tasks for Opendkim Ansible role
+
+- fail:
+    msg: "Bailing out. This role requires 'dkim_admin_email' to have an email address as value"
+  when: dkim_admin_email is not defined
+
+- fail:
+    msg: "Bailing out. This role requires 'dkim_domains' to be defined as a list of domains."
```

Comment:
Inconsistent value. Dot here and not upper.

Reply:
"bailing out, this role requires..." is what you mean?